jbpm-console-ng-generic/jbpm-console-ng-generic-client/src/main/java/org/jbpm/console/ng/gc/client/menu/ServerTemplateSelectorMenuBuilder.java

```diff
@@ -47,16 +47,31 @@
     @Inject
     private Event<ServerTemplateSelected> serverTemplateSelectedEvent;
 
+
+    public ServerTemplateSelectorMenuBuilder(){
+
+    }
+
+    public ServerTemplateSelectorMenuBuilder(Caller<SpecManagementService> specManagementService, ServerTemplateSelectorView view ){
+        this.specManagementService =specManagementService;
```

Comment:
I started to use setter methods with `@Inject`. I found it to be more flexible than changing the constructor. Mockito autocratically injects the mocks it knows and for the caller you just use the setter method in the setup instead.